### PR TITLE
Replace ArrayList for primitive array in copyArray method

### PR DESCRIPTION
**Pull request by avargas-nearsoft**
at org.junit.runner.JUnitCommandLineParseResult
According to https://github.com/junit-team/junit4/issues/1271

### CODE REVIEW

File: src/main/java/org/junit/runner/JUnitCommandLineParseResult.java [hunk 1]
```diff
@@ -85,13 +85,11 @@ private void parseArgs(String[] args) {
     }
 
     private String[] copyArray(String[] args, int from, int to) {
-        ArrayList<String> result = new ArrayList<String>();
-
+        String [] result = new String[to-from];
```
Comment (kcooney) [inline src/main/java/org/junit/runner/JUnitCommandLineParseResult.java]: Style-nit: we put spaces around operators like -

File: src/main/java/org/junit/runner/JUnitCommandLineParseResult.java [hunk 2]
```diff
@@ -85,13 +85,11 @@ private void parseArgs(String[] args) {
     }
 
     private String[] copyArray(String[] args, int from, int to) {
-        ArrayList<String> result = new ArrayList<String>();
-
+        String [] result = new String[to-from];
         for (int j = from; j != to; ++j) {
-            result.add(args[j]);
+            result [j-from] = args[j];
```
Comment (kcooney) [inline src/main/java/org/junit/runner/JUnitCommandLineParseResult.java]: We don't put spaces when indexing into an array. Please change to:

    result[j - from] = args[j];

**Comment by avargas-nearsoft** [comment]
Updated.

**Comment by marcphilipp** [comment]
Thanks!

**Pull request merged**